src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SourcelinkTests.cs

```diff
@@ -26,24 +26,37 @@ public class SourcelinkTests : SmokeTests
     [Fact]
     public void VerifySourcelinks()
     {
-        if (Directory.Exists(SourcelinkRoot))
+        try
         {
-            Directory.Delete(SourcelinkRoot, true);
-        }
-        Directory.CreateDirectory(SourcelinkRoot);
+            if (Directory.Exists(SourcelinkRoot))
+            {
+                Directory.Delete(SourcelinkRoot, true);
+            }
+            Directory.CreateDirectory(SourcelinkRoot);
 
-        IList<string> failedFiles = ValidateSymbols(ExtractSymbolsPackages(GetAllSymbolsPackages()), InitializeSourcelinkTool());
+            string symbolsRoot = Directory.CreateDirectory(Path.Combine(SourcelinkRoot, "symbols")).FullName;
+            Utilities.ExtractTarball(
```

Comment:
I think it might be helpful to call out in a comment that the dotnet-sdk-symbols content is derived from dotnet-symbols-* therefore no validation is needed on it.

Reply:
Fixed in https://github.com/dotnet/installer/pull/17477/commits/b793194293c24b745ac64566dfa46b93957a1bc3